<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordEditDialog</class>
 <widget class="QDialog" name="WordEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>197</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource.qrc">
    <normaloff>:/icons/images/ui-combo-box-edit.png</normaloff>:/icons/images/ui-combo-box-edit.png</iconset>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Signed:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signedLineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
    </font>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="unsignedLineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
    </font>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Unsigned:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hexLineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>40</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bytes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Expression:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="expressionLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>165</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnOk">
      <property name="text">
       <string>&amp;OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>&amp;Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ASCII:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="asciiLineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>expressionLineEdit</tabstop>
  <tabstop>hexLineEdit</tabstop>
  <tabstop>signedLineEdit</tabstop>
  <tabstop>unsignedLineEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>WordEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>WordEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
